packages/@lwc/integration-karma/test/light-dom/slot-fowarding/slots/reactivity/index.spec.js

```diff
@@ -7,14 +7,14 @@ describe('light DOM slot forwarding reactivity', () => {
     let nodes;
     let lightContainer;
 
-    beforeAll(() => {
+    beforeEach(() => {
```

Comment:
In `setup.mjs`, we clear the children after every test. So we need to update this to create the expected scenario for these tests.